dev/ci/presubmits/validate-generated

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
We should just remove `dev/ci/presubmits/validate-generated-types` since the new check already covers type validation.